recipes/python-chromedriver-binary/bld.bat

```
@@ -1,5 +1,5 @@
-curl -O https://chromedriver.storage.googleapis.com/2.36/chromedriver_win32.zip
-7za x chromedriver.zip -ochromedriver
+python download-chromedriver
```

Should be `python %RECIPE_DIR%\download-chromedriver.py`

---

Also after this line add `if errorlevel 1 exit 1` so it fails early